recipes/restinio/all/conanfile.py

```diff
@@ -49,14 +49,17 @@ def configure(self):
 
     def requirements(self):
         self.requires("http_parser/2.9.4")
-        self.requires("fmt/6.2.1")
+        self.requires("fmt/7.0.3")
         self.requires("expected-lite/0.4.0")
         self.requires("optional-lite/3.2.0")
-        self.requires("string-view-lite/1.3.0")
+        self.requires("string-view-lite/1.4.0")
         self.requires("variant-lite/1.2.2")
 
         if self.options.asio == "standalone":
-            self.requires("asio/1.16.1")
+            if tools.Version(self.version) > "0.6.9":
```

Comment:
It won't use `asio/1.17.0` for the new version we are adding. Typo? 

... otherwise we should remove this, as we cannot know if `>0.6.9` builds with `asio/1.17.x`.

Reply:
Good Catch!